ts/src/lbank2.ts

```diff
@@ -278,37 +292,90 @@ export default class lbank2 extends Exchange {
         });
     }
 
+    async fetchTime (params = {}) {
+        /**
+         * @method
+         * @name lbank2#fetchTime
+         * @description fetches the current integer timestamp in milliseconds from the exchange server
+         * @see https://www.lbank.info/en-US/docs/index.html#get-timestamp
+         * @see https://www.lbank.com/en-US/docs/contract.html#get-the-current-time
+         * @param {object} [params] extra parameters specific to the lbank2 api endpoint
+         * @returns {int} the current integer timestamp in milliseconds from the exchange server
+         */
+        let type = undefined;
+        [ type, params ] = this.handleMarketTypeAndParams ('fetchTime', undefined, params);
+        let response = undefined;
+        if (type === 'swap') {
+            response = await this.contractPublicGetCfdOpenApiV1PubGetTime (params);
+        } else {
+            response = await this.spotPublicGetTimestamp (params);
+        }
+        //
+        // spot
+        //
+        //     {
+        //         "result": "true",
+        //         "data": 1691789627950,
+        //         "error_code": 0,
+        //         "ts": 1691789627950
+        //     }
+        //
+        // swap
+        //
+        //     {
+        //         "data": 1691789627950,
+        //         "error_code": 0,
+        //         "msg": "Success",
+        //         "result": "true",
+        //         "success": true
+        //     }
+        //
+        return this.safeInteger (response, 'data');
+    }
+
     async fetchMarkets (params = {}) {
         /**
          * @method
          * @name lbank2#fetchMarkets
          * @description retrieves data on all markets for lbank2
          * @see https://www.lbank.com/en-US/docs/index.html#trading-pairs
+         * @see https://www.lbank.com/en-US/docs/contract.html#query-contract-information-list
          * @param {object} [params] extra parameters specific to the exchange api endpoint
          * @returns {object[]} an array of objects representing market data
          */
-        // needs to return a list of unified market structures
-        const response = await this.publicGetAccuracy ();
-        const data = this.safeValue (response, 'data');
-        //      [
-        //          {
-        //              symbol: 'snx3s_usdt',
-        //              quantityAccuracy: '2',
-        //              minTranQua: '0.01',
-        //              priceAccuracy: '6'
-        //          }
-        //     ]
+        const spotMarkets = await this.fetchSpotMarkets (params);
+        const swapMarkets = await this.fetchSwapMarkets (params);
```

Comment:
@Dan-krm we should always use Promise.all to resolve both promises "in parallel"